.github/workflows/docker.yml

```
@@ -0,0 +1,68 @@
+name: Build RDMP Docker Image
+on:
+  push:
+
+jobs:
+  build:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+    
```

```suggestion
    permissions:
      contents: read
      packages: write

```
Dunno if this helps..... is the org set to default to read-only on the CI token?